webApp-plivo/public/js/customclient.js

```diff
@@ -153,6 +153,11 @@ function onDtmfReceived(dtmfData) {
 	console.log("Received dtmf is ", dtmfData.tone)
 }
 
+function remoteAudioStatus(hasAudio) {
+	console.log("Received remoteAudioStatus is ", hasAudio)
```

Comment:
show a alert when this event is received. Just like when mediaMetrics event is received